### ISSUE TITLE
Realistic Item Drops

### ISSUE THREAD

**Issue by Silabear** [mc-datapacks/review-tracker#85]
Project Link: https://www.planetminecraft.com/data-pack/better-item-dropping/

GitHub Link: https://github.com/Silabear/Realistic-Item-Drops

Discord Username: Silabear#0530

Certified Compatibility ✅ + Style Guideline Followed 📚

**Comment by oOBoomberOo** [comment]
A bounty of $10 has been added to this review. You will be rewarded if you choose to review this datapack.

*Keep in mind that I will be expecting a high-quality review with this bounty.*

**Comment by EstEarth202** [comment]
- **Request review:** If you have, please include every website where your datapack is published. It doesn't have to be just GitHub or Planetminecraft. And if you're on the Minecraft Datapacks discord server, add your `Username#0000` as well.
   - **Review type:** please also specify the type of review you require.
     - Certified Compatibility ✅ Only
     - Certified Compatibility ✅ + Style Guideline Followed 📚 (Optional)

**Comment by Silabear** [comment]
Yeah, my Datapack is only on PlanetMinecraft. I believe I haven't followed the style guidelines as my pack has a message sent on /reload, but other than that I believe that it is alright. My username is Silabear#0530, for some reason I was having trouble sending it properly :D

**Comment by EstEarth202** [comment]
Would you like some advice on the style guidelines?  You can fix it before I review.

**Comment by Silabear** [comment]
Sure, I'll try and remake it. At the time, I didn't really pay attention to it. There is an advancement, but I left the reload message. Hopefully I'll do this quickly :D
